pkgs/development/libraries/gdal/default.nix

```diff
@@ -20,14 +20,13 @@ composableDerivation.composableDerivation {} (fixed: rec {
 
   hardeningDisable = [ "format" ];
 
-  # Don't use optimization for gcc >= 4.3. That's said to be causing segfaults.
   # Unset CC and CXX as they confuse libtool.
-  preConfigure = "export CFLAGS=-O0 CXXFLAGS=-O0; unset CC CXX";
```

Comment:
@LnL7: The comment in the explanation is about unsetting $CC and $CXX, which I did leave as is. I just removed the -O0 flags.